install/kubernetes/helm/istio/charts/galley/templates/service.yaml

```diff
@@ -17,5 +17,7 @@ spec:
     name: http-monitoring
   - port: 9901
     name: grpc-mcp
+  - port: 15019
+    name: grpc-tls-mcp
```

Comment:
Does this mean the sidecar is only used for incoming mTLS connections? Don't we also want the envoy stats for clear text?